src/Inventory/Asset/Software.php

```diff
@@ -94,6 +94,31 @@ public function prepare(): array
         $with_manufacturer = [];
         $without_manufacturer = [];
 
+        //put OS as software
+        $main_asset_raw_data = $this->getMainAsset()->getRawData();
+        if (
+            property_exists($main_asset_raw_data, 'content')
+            && property_exists($main_asset_raw_data->content, 'operatingsystem')
```

Comment:
OperatingSystem data is already available as `$this->extra_data[OperatingSystem::class]`